Detections/SecurityAlert/AVTarrask.yaml

```diff
@@ -0,0 +1,41 @@
+id: 1785d372-b9fe-4283-96a6-3a1d83cabfd1
+name: AV detections related to Tarrask malware
+description: |
+  'This query looks for Microsoft Defender AV detections related to Tarrask malware. In Microsoft Sentinel the SecurityAlerts table 
+   includes only the Device Name of the affected device, this query joins the DeviceInfo table to clearly connect other information such as Device group, ip, logged on users etc. 
+   This would allow the Microsoft Sentinel analyst to have more context related to the alert, if available.
+   Reference: https://www.microsoft.com/security/blog/2022/04/12/tarrask-malware-uses-scheduled-tasks-for-defense-evasion/'
+severity: High
+requiredDataConnectors:
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - SecurityAlert
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Persistence
+relevantTechniques:
+  - T1053
+query: |
+  let Tarrask_threats = dynamic(["HackTool:Win64/Tarrask!MS", "HackTool:Win64/Ligolo!MSR", "Behavior:Win32/ScheduledTaskHide.A"]);
+  DeviceInfo
+  | extend DeviceName = tolower(DeviceName)
+  | join ( SecurityAlert
```

Comment:
You might want to make this a rightouter join type in case users don't have DeviceInfo data onboarded - that way it will still return results.